internal/api/v1/stage.go

```diff
@@ -62,7 +64,7 @@ func (hc ApplicationsController) Stage(w http.ResponseWriter, r *http.Request) A
 		return singleNewError("name parameter from URL does not match name param in body", http.StatusBadRequest)
 	}
 
-	if req.Instances < 0 {
+	if req.Instances != nil && *req.Instances < 0 {
 		return APIErrors{NewAPIError(
```

Comment:
Ditto `BadRequest` utility usable here ?

Reply:
I do want to change how errors are handled in the API and have some ideas, but changing line 68 is not part of this PR.